persistence/sql/persister_schema.go

```diff
@@ -0,0 +1,27 @@
+package sql
+
+import (
+	"errors"
+	"github.com/gobuffalo/pop"
+	"github.com/gofrs/uuid"
+	"github.com/ory/kratos/schema"
+	"github.com/ory/x/sqlcon"
+)
+
+func (p *Persister) GetSchema(id uuid.UUID) (*schema.JsonSchema, error) {
+	var s schema.JsonSchema
+	if err := p.c.Find(&s, id); err != nil {
+		return nil, sqlcon.HandleError(err)
+	}
+	return &s, nil
+}
+
+func (p *Persister) CreateSchema(s schema.JsonSchema) error {
+	if s.Url == "" {
+		return errors.New("URL must be provided")
```

Comment:
We can add more context here. We have the herodot package which allows us to return errors with extra info - such as HTTP status code. This mistake would case a 400 Bad Request (because the client sent invalid payload):

```
return errors.WithStack(herodot.ErrBadRequest.WithReason("The schema is missing the URL property."))
```